fe1-web/src/features/lao/components/LaoProperties.tsx

```diff
@@ -42,7 +39,7 @@ const LaoProperties = ({ url }: IPropTypes) => {
           <ListCollapsibleIcon isOpen={toggleChildrenVisible} />
         </TouchableOpacity>
 
-        {toggleChildrenVisible && lao && renderProperties(lao, url)}
+        {toggleChildrenVisible && lao && <Properties lao={lao} url={url} />}
```

Comment:
Outside the scope of this PR (please!), but it seems that a few things are off here.
1) the naming is off.  The `LaoProperties` component is only doing toggling , whereas `Properties` is actually doing the displaying.
2) If i'm not mistaken, the URL information is not available as part of the LAO. In any case it doesn't make sense to get the URL as a parameter but the LAO from the storage.

Reply:
Yes, I totally agree! This component has actually already been updated in a different commit, unfortunately in one that is already part of the PR as well 😅😅

Reply:
But I have to update it once more as there was an issue during the pop party today 😅